README.md

```diff
@@ -103,5 +103,13 @@ In this case, the result would be:
 }
 ```
 
+## Known Issues
```

Comment:
a known issue implies that there is a understood bug. I don't think this is a bug

Reply:
You totally right.